packages/picker/src/Picker.ts

```diff
@@ -423,19 +435,7 @@ export class PickerBase extends SizedMixin(Focusable, { noDefaultSize: true }) {
                       `
                     : nothing}
                 ${when(this.pending, () => {
```

Comment:
Could this `when` statement be delegated to the controller? and then here we would only need `this.pendingStateController.renderPendingState()`?

Reply:
Makes Sense